Do-ItTests/DoItTableViewControllerTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  DoItTableViewControllerTests.swift
+//  Do-ItTests
+//
+//  Created by Aaron on 2/19/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import XCTest
+// @testable import DoItTableViewController
+@testable import Do_It
+
+class DoItTableViewControllerTests: XCTestCase {
```

Comment:
Once searching is pulled out into a separate class, let's add some tests for it.